src/js/blocksetting123.js

```diff
@@ -154,6 +157,91 @@ function assessPerformance(prediction, outcome) {
 /***
  *practice block n < n_TrialPractice + 1
  */
+ function practice_block0(timeline, jsPsych) {
+  let n_TrialPractice1 = 10;
+  let counterP_1 = 0;
+  let c1 = 0;
+  let jitters_1 = GenerateJitter(n_TrialPractice1, n_MaxJitter);
+  //let jitters_1 = 3; //ensure there are no changepoints in the practice
+  let trial_type_label = 'practice';
+
+  for (let n = 1; n < n_TrialPractice1 + 1; n++) {
+    const colorStyleP = colorP0;
+    var x1;
+    let prediction;
+    let outcome;
+    let mean;
+    counterP_1++;
+    if (counterP_1 <= n_SamePosition + jitters_1[c1]) {
+      // counterP_1 = counterP_1;
+    }
+    if (counterP_1 > n_SamePosition + jitters_1[c1]) {
+      counterP_1 = Math.mod(counterP_1, n_SamePosition + jitters_1[c1]);
+      c1++;
+    }
+    if (counterP_1 === 1) {
+      x1 = 270;//nums2_1[n];
+    }
+    if (counterP_1 !== 1) {
+      // x1 = x1
+    }
+    // make task slightly easier for practicing with lower noise stdev -- CHANGED SO NOT TRUE (REALISTIC TO TASK)
+    outcome = Math.mod(normalRandomScaled(x1, 20), 360);
```

Comment:
This can be simplified significantly for the sake of readability and maintenance in the future.